### PR TITLE
Add SEO page, "Update Browser" (#6710) 

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
* Page is live before end of January 2019
* New page is at URL /technology/update-your-browser/

## Issue / Bugzilla link
#6710

## Issue / Bugzilla link
- Copy pasta errors
- Correct URL & meta data
https://bedrock-demo-achurchwell.oregon-b.moz.works/en-US/technology/update-your-browser/

### CODE REVIEW

**Comment by amychurchwell** [comment]
Waiting for stakeholder demo approval

File: bedrock/mozorg/templates/mozorg/update-browser.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      <h1 class="mzp-c-article-title">{{ _('Why Firefox?') }}</h1>
+      <p class="mzp-c-article-intro">
+        {% trans %}
+          Firefox is independent and a part of the not-profit Mozilla, which fights for your online rights, keeps corporate powers in check and makes the internet accessible to everyone, everywhere. We believe the internet is for people, not profit. Unlike other companies, we don’t sell access to your data. You’re in control over who sees your search and browsing history. All that and exceptional performance too.
+        {% endtrans %}
+      </p>
+      <h2 class="c-section-title">{{ _('How do I update?') }}</h2>
+      <p>
+      {% trans %}
+        Most major browsers update automatically, which means that when a new version is available, your system will download and install it for you. So that’s one less thing you have to worry about keeping on top of. First find out which version of your browser you’re on. Then, if you have your auto update switched off, here’s how to Firefox, Edge, Chrome, Opera or Safari.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/mozorg/templates/mozorg/update-browser.html]: @alexgibson 
What do you think about -- "First find out which version of your browser you’re on. Then, if auto update is switched off, here’s how to manually update [Firefox], [Edge], [Chrome], [Opera] or [Safari]." ?
[] = links

[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/update-browser.html]: Works for me 👍

File: bedrock/mozorg/templates/mozorg/update-browser.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+     <h2 class="c-section-title">{{ _('The best privacy') }}</h2>
+     <p>
+     {% trans privacy="https://blog.mozilla.org/firefox/make-your-firefox-browser-a-privacy-superpower-with-these-extensions/"%}
+        We are passionate about user privacy. With each new release we give you more ways to control who sees and accesses your personal browsing data. Tracking protection, private browsing and <a href="{{ privacy }}">powerful privacy extensions</a> all work together to make sure your private browsing information stays yours.
+     {% endtrans %}
+     </p>
+
+     <h2 class="c-section-title">{{ _('New features') }}</h2>
+     <p>
+     {% trans accessibility="https://support.mozilla.org/en-US/kb/accessibility-features-firefox-make-firefox-and-we", extensions="https://blog.mozilla.org/firefox/its-a-new-firefox-multi-tasking-extension-side-view/" %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/update-browser.html]: These links could have the `/en-US/` locale removed, so they display in the visitors preferred language. Not a blocker though, seeing as this page is en-US only.

**Comment by ejregithub** [comment]
@alexgibson - this is good to go, stakeholder approved. I removed the DNM tag.

**Pull request merged**